akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -459,9 +459,16 @@ private[remote] abstract class ArteryTransport(_system: ExtendedActorSystem, _pr
 
     startRemoveQuarantinedAssociationTask()
 
-    log.info(
-      "Remoting started with transport [Artery {}]; listening on address [{}] with UID [{}]",
-      settings.Transport, localAddress.address, localAddress.uid)
+    if (localAddress.address == bindAddress.address)
+      log.info(
+        "Remoting started with transport [Artery {}]; listening on address [{}] with UID [{}]",
+        settings.Transport, bindAddress.address, bindAddress.uid)
+    else {
+      log.info(
+        s"Remoting started with transport [Artery ${settings.Transport}]; listening on address [{}] with UID [{}] and bound to [{}] with UID [{}]",
```

Comment:
The UID will be the same but if you're sick of making small changes for this PR then we can merge and fix later :)

Reply:
no problems, I wondered about that but wasn't sure. thanks!